internal/ci_setup.sh

```diff
@@ -1,5 +1,4 @@
-$GOBIN/natsc request -s $NATS_URI -t 5 -r 99 'group.set' '{"id":"1","name": "ci_admin"}'
```

Comment:
We may need to implement the same changes on the [setup](https://github.com/ernestio/ernest/blob/master/setup#L107) script